packages/nouns-webapp/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "jsx": "react-jsx"
   },
   "include": [
-    "src"
+    // "src"
```

Comment:
reminder to uncomment this  -- unfortunately its the only way I'm able to run the project locally without fixing like 100 typescript warnings .